proxy.config.json

```
@@ -97,5 +97,9 @@
   "urlShortener": "",
   "contactEmail": "",
   "csrfProtection": true,
-  "plugins": []
+  "tls": {
+    "enabled": false,
```

```suggestion
    "enabled": true,
```

For the purposes of a good (and secure!) default experience, let's keep TLS enabled by default since there's already self-signed certs shipped with the repo.